### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by LixiangHello** [Jupyter-contrib/jupyter_nbextensions_configurator#160]
### **I don't known how to do in the case.**

In Configurable nbextensions of jupyter page.

_[jupyter_nbextensions_configurator] error loading scratchpad/main_

>  This is my `jupyter nbextension list`
>   notebook section
>         nbextensions_configurator/config_menu/main enabled
>         - Validating: problems found:
>           - require?  X nbextensions_configurator/config_menu/main
>         contrib_nbextensions_help_item/main enabled
>         - Validating: ok
>         tree section
>         nbextensions_configurator/tree_tab/main enabled
>         - Validating: problems found:
>           - require?  X nbextensions_configurator/tree_tab/main
>         ...

### **And I have got this error.**
  [W 14:14:53.991 NotebookApp] 404 GET /nbextensions/pydeck/extension.js?v=20230531141359 (::1) 8.000000ms referer=http://localhost:8888/notebooks/D86/test.ipynb
  [W 14:14:54.955 NotebookApp] 404 GET /nbextensions/widgets/notebook/js/extension.js?v=20230531141359 (::1) 3.570000ms referer=http://localhost:8888/notebooks/D86/test.ipynb

**Comment by LixiangHello** [comment]
Yes. First, I completely reintall conda, but failed! Then, I delete any fold about jupyter from my conputer. Yes, it's OK now. You can try.

**Comment by SDaoer** [comment]
Thanks for your reply! This did not work for me. I almost find the problem, but i could not solve it. It seems that the extension loading path has duplicated item.
